src/app/login-register/service/user-register.service.spec.ts

```diff
@@ -0,0 +1,16 @@
+import { TestBed } from '@angular/core/testing';
+
+import { UserRegisterService } from './user-register.service';
+
+describe('UserRegisterService', () => {
+  let service: UserRegisterService;
+
+  beforeEach(() => {
+    TestBed.configureTestingModule({});
+    service = TestBed.inject(UserRegisterService);
+  });
+
+  it('should be created', () => {
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/MyCookieBook/CookieBookFE/pullRequest?prid=7552947)
```suggestion
  it("should be created", () => {
```